lib/ClangImporter/ImportType.cpp

```diff
@@ -929,6 +929,8 @@ namespace {
     SUGAR_TYPE(Adjusted)
     SUGAR_TYPE(SubstTemplateTypeParm)
     SUGAR_TYPE(Elaborated)
+    SUGAR_TYPE(Using)
```

Comment:
Not sure the best person to ask about this one - @zoecarver since it's C++ only?